solr/core/src/java/org/apache/solr/handler/ReplicationHandler.java

```diff
@@ -764,12 +765,8 @@ public FileInfo(long lasmodified, String name, long size, long checksum) {
       this.checksum = checksum;
     }
 
-    Map<String, Object> getAsMap() {
-      Map<String, Object> map = new HashMap<>();
-      map.put(NAME, name);
-      map.put(SIZE, size);
-      map.put(CHECKSUM, checksum);
-      return map;
+    CoreReplicationAPI.FileMetaData getAsMap() {
```

Comment:
[-1] We probably need to rename this method, since it's no longer returning a map 😛